src/styles/base/_variables.scss

```diff
@@ -12,30 +12,29 @@ $mc-color-light:             #fff !default;
 $mc-color-dark:              #000 !default;
 
 // Colors
+$mc-color-gray-100:           #191c21 !default;
+$mc-color-gray-200:           #272c33 !default;
+$mc-color-gray-300:           #31333b !default;
+$mc-color-gray-400:           #596170 !default;
+$mc-color-gray-500:           #949aa4 !default;
+$mc-color-gray-600:           #dbdde1 !default;
+
 $mc-color-primary:           #c83232 !default;
 $mc-color-primary-hover:     #d63636 !default;
 $mc-color-primary-active:    #b22b2b !default;
 
-$mc-color-secondary:         #232325 !default;
-$mc-color-secondary-hover:   #2c2c2e !default;
-$mc-color-secondary-active:  #1f1f21 !default;
+$mc-color-secondary:         $mc-color-gray-200 !default;
+$mc-color-secondary-hover:   $mc-color-gray-300 !default;
+$mc-color-secondary-active:  $mc-color-gray-100 !default;
 
-$mc-color-tertiary:          #75787f !default;
-$mc-color-tertiary-hover:    #90949d !default;
+$mc-color-tertiary:          $mc-color-gray-400 !default;
+$mc-color-tertiary-hover:    $mc-color-gray-500 !default;
```

Comment:
eventually, we should prolly have a var for tertiary active as well

Reply:
Once it's comped!